default.nix

```diff
@@ -413,8 +414,8 @@ in rec {
       exe = serverOn mainProjectOut dummyVersion;
       server = args@{ hostName, adminEmail, routeHost, enableHttps, version, module ? serverModules.mkBaseEc2, redirectHosts ? [], configHash ? "" }:
         server (args // { exe = linuxExe version; });
-      obelisk = import (base' + "/.obelisk/impl") {};
-    };
+      obelisk = import (base' + "/.obelisk/impl") { inherit system profiling iosSdkVersion config terms reflex-platform-func useGHC810 nixpkgsOverlays ; };
```

Comment:
I hope this is cromulent, does it make sense to pass these same arguments along?